lectures/getting_started_julia/getting_started.md

```diff
@@ -476,7 +476,7 @@ Now we `Shift + Enter` to produce this
 #### Inserting Unicode (e.g. Greek letters)
 
 Julia supports the use of [unicode characters](https://docs.julialang.org/en/v1/manual/unicode-input/)
-such as `α` and `β` in your code.
+such as `alpha` and `beta` in your code.
```

Comment:
This should go back to the  old:
```
α` and `β` in your code.   
```

Because it is literally talking about unicode characters.